mechanics/tools/gdocs-migrate/main.go

```diff
@@ -26,28 +28,49 @@ func main() {
 	}
 
 	drives := map[string]string{
-		"Knative Community [old]": KnativeCommunityDriveId,
-		"Knative Team":            KnativeTeamDriveId,
+		"Knative Community Drive": KnativeCommunityDriveID,
+		"Knative Team Drive":      KnativeTeamDriveID,
 	}
 
+	forest := NewForest()
 	for name, id := range drives {
-		// List files from Knative Community Drive.
-		r, err := srv.Files.List().PageSize(10).
-			Corpora("drive").
-			IncludeItemsFromAllDrives(true).
-			SupportsAllDrives(true).
-			DriveId(id).
-			Fields("nextPageToken, files(id, name)").Do()
-		if err != nil {
-			log.Fatal("Unable to retrieve files:", err)
-		}
-		fmt.Printf("---------------------------\n%s files:\n---------------------------\n", name)
-		if len(r.Files) == 0 {
-			fmt.Println("No files found.")
-		} else {
+		fmt.Println("Loading directory from ... ", name)
+		forest.Add(File{
+			id:       id,
+			name:     name,
+			isFolder: true,
+		})
+		var pageToken *string
+		for pageToken == nil || *pageToken != "" {
```

Comment:
If you're handling `pageToken == ""`, any reason pageToken needs to be a `*string` and not a `string`?

Reply:
I use `nil` for first page, and "" for last page.